dev/MRTCore/mrt/Core/unittests/MrmTests.cpp

```diff
@@ -343,58 +370,58 @@ TEST_CLASS(BasicTest)
     TEST_METHOD(EnumerateResourceMap)
     {
         MrmManagerHandle resourceManager;
-        Assert::AreEqual(MrmCreateResourceManager(L".\\resources.pri", &resourceManager), S_OK);
+        VERIFY_ARE_EQUAL(MrmCreateResourceManager(L".\\resources.pri", &resourceManager), S_OK);
 
         // Enumerate under Microsoft.UI.Xaml/Resources
         MrmMapHandle childResourceMap;
-        Assert::AreEqual(MrmGetChildResourceMap(resourceManager, nullptr, L"Microsoft.UI.Xaml", &childResourceMap), S_OK);
+        VERIFY_ARE_EQUAL(MrmGetChildResourceMap(resourceManager, nullptr, L"Microsoft.UI.Xaml", &childResourceMap), S_OK);
 
         MrmMapHandle childChildResourceMap;
-        Assert::AreEqual(MrmGetChildResourceMap(resourceManager, childResourceMap, L"Resources", &childChildResourceMap), S_OK);
+        VERIFY_ARE_EQUAL(MrmGetChildResourceMap(resourceManager, childResourceMap, L"Resources", &childChildResourceMap), S_OK);
 
         UINT32 count;
-        Assert::AreEqual(MrmGetResourceCount(resourceManager, childChildResourceMap, &count), S_OK);
-        Assert::AreEqual<UINT32>(count, 78);
+        VERIFY_ARE_EQUAL(MrmGetResourceCount(resourceManager, childChildResourceMap, &count), S_OK);
+        VERIFY_ARE_EQUAL(count, static_cast<UINT32>(78));
 
         MrmType resourceType;
         wchar_t* resourceString = nullptr;
         wchar_t* resourceName = nullptr;
         MrmResourceData resourceData {};
         for (UINT32 i = 0; i < count; i++)
         {
-            Assert::AreEqual(MrmLoadStringOrEmbeddedResourceByIndex(resourceManager, nullptr, childChildResourceMap, i, &resourceType, &resourceName, &resourceString, &resourceData), S_OK);
+            VERIFY_ARE_EQUAL(MrmLoadStringOrEmbeddedResourceByIndex(resourceManager, nullptr, childChildResourceMap, i, &resourceType, &resourceName, &resourceString, &resourceData), S_OK);
 
-            Assert::IsTrue(resourceType == MrmType_String);
-            Assert::IsNull(resourceData.data);
-            Assert::AreEqual(resourceData.size, 0u);
+            VERIFY_IS_TRUE(resourceType == MrmType_String);
+            VERIFY_IS_NULL(resourceData.data);
+            VERIFY_ARE_EQUAL(resourceData.size, 0u);
 
             switch (i)
             {
             case 0:
             {
-                Assert::AreEqual(resourceName, L"AutomationNameAlphaSlider");
-                Assert::AreEqual(resourceString, L"Opacity");
+                VERIFY_ARE_EQUAL(wcscmp(resourceName, L"AutomationNameAlphaSlider"), 0);
+                VERIFY_ARE_EQUAL(wcscmp(resourceString, L"Opacity"), 0);
                 break;
             }
 
             case 1:
             {
-                Assert::AreEqual(resourceName, L"AutomationNameAlphaTextBox");
-                Assert::AreEqual(resourceString, L"Opacity");
+                VERIFY_ARE_EQUAL(wcscmp(resourceName, L"AutomationNameAlphaTextBox"), 0);
```

Comment:
We have this often enough that a helper macro might be beneficial. That would also have the ability to print the expected versus actual string, which the current macro would not do.

Reply:
Good idea! I'll add a helper for comparing strings.